src/ProjectTemplates/Web.ProjectTemplates/content/Worker-CSharp/.template.config/vs-2017.3.host.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json.schemastore.org/vs-2017.3.host",
+  "name": {
+    "text": "Worker Service",
+    "package": "{0CD94836-1526-4E85-87D3-FB5274C5AFC9}",
+    "id": "1011"
+  },
+  "description": {
+    "text": "An empty project template for creating a worker service. This template does not have any content in it.",
+    "package": "{0CD94836-1526-4E85-87D3-FB5274C5AFC9}",
+    "id": "1012"
+  },
+  "order": 100,
+  "icon": "vs-2017.3/Empty.png",
+  "learnMoreLink": "https://go.microsoft.com/fwlink/?LinkID=784883",
+  "uiFilters": [ "oneaspnet" ],
+  "supportsDocker": true,
+  "legacyTemplateIdentity": "Microsoft.NetCore.CSharp.EmptyWorker",
```

Comment:
All this auth and port stuff is presumably not required.

Reply:
I'm not messing with the VS template until we can test it in VS. This is a copy of Empty.